### PR TITLE
samples: crypto: Added missing spake2+ board config files

### PR DESCRIPTION
**Pull request by magnev**
Required for spake2+ to work since Oberon must be enabled for devices with CryptoCell, and Mbed TLS heap must be set.

### CODE REVIEW

File: samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
```
Comment (tomi-font) [inline samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf:5]: Suggest removing that `#` to mark a clear distinction between header and start of configuration.

File: samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# Oberon is required for spake2+
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
+CONFIG_PSA_CRYPTO_DRIVER_CC3XX=y
+
+# Mbedtls configuration
```
Comment (tomi-font) [inline samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf:10]:
```suggestion
# Mbed TLS configuration
```
Or maybe rather say something like `needed for spake2+`?

**Review by tomi-font — APPROVED** [review]

File: samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf (hunk 1 above)
Comment (magnev) [reply samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf:5]: If I fix this we should do it for all crypto samples to have the config file structure aligned.
I don't want to do that in this bugfix commit.

File: samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf (hunk 2 above)
Comment (magnev) [reply samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf:10]: Same as the above comment. Let's not refactor comments now

**Review by stephen-nordic — APPROVED** [review]

**Review by hellesvik-nordic — APPROVED** [review]

**Pull request merged**